app/src/main/java/de/danoeh/antennapod/fragment/preferences/MainPreferencesFragment.java

```diff
@@ -87,14 +87,16 @@ private void setupMainScreen() {
         });
         findPreference(PREF_ABOUT).setOnPreferenceClickListener(
                 preference -> {
-                    getParentFragmentManager().beginTransaction().replace(R.id.content, new AboutFragment())
+                    getParentFragmentManager().beginTransaction()
+                            .replace(R.id.settings_container, new AboutFragment())
```

Comment:
I think there are more places left where we use `R.id.content`. The about page, for example.